packages/doctype-caip10-link/src/caip10-link-doctype.ts

```diff
@@ -36,80 +22,107 @@ const throwReadOnlyError = (): Promise<void> => {
 @DoctypeStatic<DoctypeConstructor<Caip10LinkDoctype>>()
 export class Caip10LinkDoctype extends Doctype {
 
+    static DOCTYPE_NAME = 'caip10-link'
+    static DOCTYPE_ID = 1
```

Comment:
Doctype ID here with TODO comment above for HandlersMap. Uff, this is quite a change coming. Would vote for talking through this.

Non-actionable for here, just in case.

Reply:
Yeah happy to talk about it.  I think we should be relying on the doctype ID over the name whenever possible within internal-only code, since that's what shows up in the protocol messages and will future-proof us in case we ever decide to rename an existing doctype.